web/app.js

```diff
@@ -874,7 +874,7 @@ const PDFViewerApplication = {
     );
   },
 
-  download() {
+  download(isFallback) {
```

Comment:
Please change this to the following, to clearly mark the parameter as optional:
```suggestion
  download(isFallback = false) {
```